### ISSUE TITLE
 module load cluster/wice/bigmem

### ISSUE THREAD

**Issue by Zhangruiqi111** [aertslab/scenicplus#425]
Hello ， when I run module load cluster/wice/bigmem , there is an error : ERROR: Unable to locate a modulefile for 'cluster/wice/bigmem' ,so where can download this file?

**Comment by SeppeDeWinter** [comment]
Hi @Zhangruiqi111 

In what context are you running this code?
This line of code is very specific to the HPC environment we are using at KULeuven, so it should not be used in general cases. Can you link me to the file/script/tutorial you are using?

Best,

Seppe

**Comment by Zhangruiqi111** [comment]
Hi @SeppeDeWinter 
I did not run this command  "module load cluster/wice/bigmem" at last, and the program ran smoothly. The problem has been solved, thank you!
Best,
Ruiqi Zhang

**Comment by yojetsharma** [comment]
> Hi @Zhangruiqi111
> 
> In what context are you running this code? This line of code is very specific to the HPC environment we are using at KULeuven, so it should not be used in general cases. Can you link me to the file/script/tutorial you are using?
> 
> Best,
> 
> Seppe

I understand that this module is part of the HPC. But is there an alternative to this? This module is currently not available on my institutional HPC cluster.

**Comment by SeppeDeWinter** [comment]
Hi @yojetsharma 

It's very likely that you don't need to run this at all.
Could you please link me to the tutorial where you find this `module load` command?

All the best,

Seppe

**Comment by yojetsharma** [comment]
Yes! It is available here: [](https://scenicplus.readthedocs.io/en/latest/human_cerebellum_ctx_db.html) 
```
Prepare fasta from consensus regions

Next we will get sequences for all the consensus peaks. We will also add 1kb of background padding, this will be used as a background sequence for cluster-buster. It is completely optional to add this padding, we have noticed that it does not affect the analyses a lot.

module load cluster/wice/bigmem
module load BEDTools/2.30.0-GCC-10.3.0

REGION_BED="/staging/leuven/stg_00002/lcb/sdewin/PhD/python_modules/pycisTopic_polars_tutorial/outs/consensus_peak_calling/consensus_regions.bed"
GENOME_FASTA="/staging/leuven/res_00001/genomes/homo_sapiens/hg38_ucsc/fasta/hg38.fa"
CHROMSIZES="/staging/leuven/res_00001/genomes/homo_sapiens/hg38_ucsc/fasta/hg38.chrom.sizes"
DATABASE_PREFIX="10x_brain_1kb_bg_with_mask"
SCRIPT_DIR="/staging/leuven/stg_00002/lcb/sdewin/PhD/python_modules/scenicplus_development_tutorial/ctx_db/create_cisTarget_databases"

${SCRIPT_DIR}/create_fasta_with_padded_bg_from_bed.sh \
        ${GENOME_FASTA} \
        ${CHROMSIZES} \
        ${REGION_BED} \
        hg38.10x_brain.with_1kb_bg_padding.fa \
        1000 \
        yes


Lmod is automatically replacing "cluster/genius/dedicated_big_bigmem" with
"cluster/wice/bigmem".


Inactive Modules:
  1) GCCcore/6.4.0                    3) ncurses/6.0-GCCcore-6.4.0
  2) libevent/2.1.8-GCCcore-6.4.0     4) tmux


Activating Modules:
  1) GCCcore/10.3.0
```
And yes, I just skipped those two lines and managed to create the cistarget databases. But I just have a query: I used the same DATA_PREFIX as in the tutorial: DATABASE_PREFIX="10x_brain_1kb_bg_with_mask" however, I am running 3 different samples with different sample_ids, this shouldn't be an issue, right?

**Comment by SeppeDeWinter** [comment]
@yojetsharma 

I see thanks!

Indeed, those lines should not be in the tutorial.

The prefix you use does not really matter ;)

All the best,

Seppe
